charts/amazon-network-policy-controller-k8s/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
```

Comment:
nit: should we add configurable annotations here too? I was thinking in terms of ever supporting IRSA via annotations